### PR TITLE
Migration to new mvr

### PR DESCRIPTION
**Pull request by PetrBainar**

### CODE REVIEW

File: spim_registration/timelapse/timelapse_registration.bsh [hunk 1]
```diff
@@ -99,19 +99,21 @@ if (channels.equalsIgnoreCase(""))
 else if (reg_process_channel.equalsIgnoreCase( "All channels" ) && channel_token.length == 1 )	
 {	
 	reg_multi_channel_string.append( "" );
-	reg_single_channel_string = "interest_points_channel_" + channels + "=" + reg_interest_points_channel + " ";
+	//reg_single_channel_string = "interest_points_channel_" + channels + "=" + reg_interest_points_channel + " ";
```
Comment (kozusznik) [inline spim_registration/timelapse/timelapse_registration.bsh:102]: consider remove unused code

File: spim_registration/timelapse/registration.bsh [hunk 2]
```diff
@@ -400,12 +413,15 @@ else if (reg_process_channel.equalsIgnoreCase( "Single channel (Select from List
 // Assembles string for Multi Channel Processing one Channel Contains the beads
 else if (reg_process_channel.equalsIgnoreCase( "Single channel (Select from List)" ) && channel_token.length > 1 && interest_token.length > 1)
 {
+/*
```
Comment (kozusznik) [inline spim_registration/timelapse/registration.bsh:416]: consider remove unused code

File: spim_registration/timelapse/registration.bsh [hunk 3]
```diff
@@ -243,6 +243,7 @@ else if (reg_process_channel.equalsIgnoreCase( "All channels" ) && channel_token
 
 	else if (type_of_detection.equalsIgnoreCase( "Difference-of-Gaussian" ) )
 	{
+		/*
```
Comment (kozusznik) [inline spim_registration/timelapse/registration.bsh:246]: consider remove unused code

File: spim_registration/timelapse/registration.bsh [hunk 4]
```diff
@@ -371,7 +380,8 @@ if (channels.equalsIgnoreCase(""))
 else if (reg_process_channel.equalsIgnoreCase( "All channels" ) && channel_token.length == 1 && interest_token.length == 1 )
 {
 	reg_multi_channel_string.append( "" );
-	reg_single_channel_string = "interest_points_channel_" + channels + "=" + reg_interest_points_channel + " ";
+//	reg_single_channel_string = "interest_points_channel_" + channels + "=" + reg_interest_points_channel + " ";
```
Comment (kozusznik) [inline spim_registration/timelapse/registration.bsh:383]: consider remove unused code

File: spim_registration/timelapse/registration.bsh [hunk 5]
```diff
@@ -383,12 +393,15 @@ else if (reg_process_channel.equalsIgnoreCase( "All channels" ) &&  ( channel_to
 // Assembles string for Multi Channel registration
 else if (reg_process_channel.equalsIgnoreCase( "All channels" ) && channel_token.length > 1 )
 {
+/*
```
Comment (kozusznik) [inline spim_registration/timelapse/registration.bsh:396]: consider remove unused code

File: spim_registration/timelapse/timelapse_registration.bsh [hunk 6]
```diff
@@ -99,19 +99,21 @@ if (channels.equalsIgnoreCase(""))
...
+	reg_single_channel_string = "interest_points_channel=" + reg_interest_points_channel + " ";
 }
 	
 // Assembles string for Multi Channel registration
 else if (reg_process_channel.equalsIgnoreCase( "All channels" ) && channel_token.length > 1 ) 
 {
-	for (int channel=0; channel < channel_token.length; channel++ )
+	/*for (int channel=0; channel < channel_token.length; channel++ )
```
Comment (kozusznik) [inline spim_registration/timelapse/timelapse_registration.bsh:109]: consider remove unused code

File: spim_registration/timelapse/timelapse_registration.bsh [hunk 7]
```diff
@@ -123,13 +125,14 @@ else if (reg_process_channel.equalsIgnoreCase( "Single channel (Select from List
 // Assembles string for Multi Channel Processing one Channel Contains the beads	
 else if (reg_process_channel.equalsIgnoreCase( "Single channel (Select from List)" ) && channel_token.length > 1 && interest_token.length > 1 )	
 {
-		for (int channel=0; channel < channel_token.length; channel++ )
+		/*for (int channel=0; channel < channel_token.length; channel++ )
```
Comment (kozusznik) [inline spim_registration/timelapse/timelapse_registration.bsh:128]: consider remove unused code

File: spim_registration/timelapse/xml_merge.bsh [hunk 8]
```diff
@@ -34,7 +34,7 @@ IJ.run("Merge Cluster Jobs",
 	"directory=" + image_file_directory + " " + 	
 	"filename_contains=job_ " + 
 	"filename_also_contains=.xml " +
-	"display " +
+//	"display " +
```
Comment (kozusznik) [inline spim_registration/timelapse/xml_merge.bsh:37]: consider remove unused code

**Comment by PetrBainar** [comment]
As per discussion with the reviewer, unused code will be subject to review within the final pull request to the original repository. The aim of this one is to merge the updated czi definition step to master so it can be tested.

**Pull request merged**